command-line-flags-for-scheduling-configuration.md

```diff
@@ -57,6 +57,12 @@ Scheduling 节点用于提供 PD 的 `scheduling` 微服务。你可以通过命
 - 默认：""
 - 如果未设置该参数，log 会默认输出到 "stderr"。如果设置了该参数，log 将输出到指定的文件。
 
+## `--name` <span class="version-mark">从 v8.3.0 版本开始引入</span>
+
++ 当前 Scheduling 节点的名字。
++ 默认：`"Scheduling-hostname"`
```

Comment:
Shall we use `scheduling` as the default name like `pd`?

Reply:
Based on the code here, do you mean modifying the code?

Reply:
yes, WDYT?

Reply:
lgtm. Maybe need to consider consistency because of data dir
https://github.com/tikv/pd/blob/f3602e31640a55b61a3b14fb3989c4a0742a2416/pkg/mcs/resourcemanager/server/config.go#L226-L234

Reply:
I prefer to remove the resource manager since it's useless for now.

Reply:
We need to cherry-pick this pr to other versions firstly, PTAL, thx~ @rleungx @qiancai 
https://github.com/pingcap/docs-cn/pull/18402

Reply:
After that, we can use lowercase to be consistent with PD.
Not sure if this can be merged in 8.3.0 though :(
https://github.com/tikv/pd/pull/8551